extend.md

```diff
@@ -54,7 +54,14 @@ load("@bazel_gazelle//:def.bzl", "DEFAULT_LANGUAGES", "gazelle_binary")
 
 gazelle_binary(
     name = "gazelle",
-    languages = DEFAULT_LANGUAGES,
+    languages = [
+        "@rules_python//gazelle", #use gazelle from rules_python
```

Comment:
Nit: in `.bzl` files, comments at the end of the line have two spaces then the `#` and then a space and then the comment. The comment should be a sentence and end with a period.

```bzl
        "@rules_python//gazelle",  # Use Gazelle from rules_python.
```

Reply:
I reformatted my comments to follow that convention. If it needs further intervention I'm happy to make more changes.